projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-column.model.ts

```diff
@@ -91,13 +95,24 @@ export class TableWidgetColumnModel {
   })
   public sort?: TableSortDirection;
 
+  @ModelInject(TableCellRendererLookupService)
+  private readonly tableCellRendererLookupService!: TableCellRendererLookupService;
+
+  @ModelInject(TableCellParserLookupService)
+  private readonly tableCellParserLookupService!: TableCellParserLookupService;
+
   public asTableColumnDef(filterAttribute?: FilterAttribute): SpecificationBackedTableColumnDef {
+    const rendererConstructor = this.tableCellRendererLookupService.lookup(this.display);
```

Comment:
Why not leave this responsibility to the table? Are we expecting table columns that are not defined through widgets to always provide a parser and renderer?